keyboards/emptystring/NQG/readme.md

```diff
@@ -0,0 +1,15 @@
+# NQG: Not Quite Gherkin
+
+![NQG](https://github.com/culturalsnow/NQG/blob/master/images/NQGLayout.jpg)
+
+NQG (Not Quite Gherkin) is a 30% ortholinear keyboard with a macro key and dedicated row for thumb keys, made by emptystring studio.
+
+Keyboard Maintainer: [SA_EndlessGame](http://twitter.com/SA_EndlessGame) on twitter  
```

Comment:
We typically have this set as a link to the maintainer's github account.

Reply:
Modified link to point to my github account instead.
Thank you!